components/button/button.css

```diff
@@ -0,0 +1,125 @@
+@import url("../../css/navbar.css");
+@import url("../../css/sidebar.css");
+
+/***** MAIN CONTENT*/
+
+.example-box {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+.btn {
+    padding: 0.5rem 1rem;
+    border-radius: 5px;
+    border: none;
+    color: var(--font-white);
+    font-family: inherit;
+    text-decoration: none;
+    font-size: inherit;
+}
+
+.btn-icon {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+}
+
+.primary {
+    background-color: #3454D1;
```

Comment:
Instead of using color hex values directly make them as variables that will come in handy.
like you had done for primary and secondary colors